ansible_collections/arista/avd/roles/eos_designs/templates/facts/switch/switch.j2

```diff
@@ -319,6 +377,10 @@ switch:
 
 {% endif %}
 
+{% if switch.mpls_tep is arista.avd.defined(true) and switch_data.combined.overlay_services_l2_only is arista.avd.defined(true) %}
```

Comment:
reconsider if you wish to support this for mpls

Reply:
Removed this.